src/oop/static_modifier/medium/q4/MainTest.java

```diff
@@ -0,0 +1,141 @@
+package oop.static_modifier.medium.q4;
+
+import global.ObjectTest;
+import global.variables.Clause;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.StringLiteral;
+import global.variables.wrappers.Optional;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static global.tools.CustomAssertions._assertArrayEquals;
+import static global.tools.CustomAssertions._assertEquals;
+import static global.utils.ArrayUtil.generateRandomArray;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class MainTest {
+    ObjectTest testClass;
+    String className = "ChemistryLab";
+    String firstDoubleName = "liquid1";
+    String secondDoubleName = "liquid2";
+    String firstIntName = "MIX_1";
+    String secondIntName = "MIX_2";
+    String arrayName = "guide";
+    String methodName = "mixFun";
+    static double initialValue = 10.5;
```

Comment:
Consider splitting this into two variables to allow for different number variation between the two variables when on the website